app/code/Magento/Sales/Api/Data/OrderAddressInterface.php

```diff
@@ -182,6 +187,13 @@ public function getCountryId();
     public function getCustomerAddressId();
 
     /**
+     * Gets the quote address ID for the order address.
+     *
+     * @return int|null Quote address ID.
+     */
+    public function getQuoteAddressId();
```

Comment:
Unfortunately you cannot add these new methods to an interface without breaking BC http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/index.html#introduction-of-a-method-to-a-class-or-interface in this case I would recommend adding a new interface to handle these new methods.

Reply:
Okey, i didnt know that policy, i will implement your solution and update the PR as soon as i a can. Thanks!! @dmanners